en,zh: add v1.5.3 release notes (#2557)
pingcap/docs-tidb-operator: en/releases/release-1.5.3.md

---

> title: TiDB Operator 1.5.3 Release Notes

Reviewer: ```suggestion
title: TiDB Operator 1.5.3 Release Notes
summary: Learn about new features and bug fixes in TiDB Operator 1.5.3.
```

---

# TiDB Operator 1.5.3 Release Notes

Release date: April 18, 2024

TiDB Operator version: 1.5.3

## New features

- Support setting `livenessProbe` and `readinessProbe` for the Discovery component ([#5565](https://github.com/pingcap/tidb-operator/pull/5565), [@csuzhangxc](https://github.com/csuzhangxc))
- Support setting `nodeSelector` for the TidbInitializer component ([#5594](https://github.com/pingcap/tidb-operator/pull/5594), [@csuzhangxc](https://github.com/csuzhangxc))

## Bug fixes

> - Fix the issue that components may be restarted when modifing the storage size of them with `configUpdateStrategy` set to `InPlace` ([#5602](https://github.com/pingcap/tidb-operator/pull/5602), [@ideascf](https://github.com/ideascf)))

Reviewer: ```suggestion
- Fix the issue that modifying the storage size of components might cause them to restart when `configUpdateStrategy` is set to `RollingUpdate` ([#5602](https://github.com/pingcap/tidb-operator/pull/5602), [@ideascf](https://github.com/ideascf))
```
